### ISSUE TITLE
Failure to write to database with no operation

### ISSUE THREAD

**Issue by jrozner** [ashirt-ops/ashirt#25]
If there are no operations loaded (application is offline) and it doesn't know of any operations ashirt will fail to save any evidence. Code blocks will silently fail, never opening the getinfo dialog, and screenshots will launch the screenshot helper but then silently fail. The images and code blocks will be saved to the directory but there will be no acknowledgment of the issue or that it was saved.

```
could not write to the database: NOT NULL constraint failed: evidence.operation_slug Unable to fetch row
```

**Comment by JoelAtDeluxe** [comment]
Yeah, this makes sense. Previously we had disabled the hotkeys when no operation was active. Do you think the proper solution is to just warn the user that they need to set an operation before they can add evidence?

**Comment by jrozner** [comment]
Ya, that seems reasonable; we probably need to build an error dialog or some sort of notification mechanism. I wonder if Qt has any type of toaster notification system that we can leverage or if a popup dialog is better? In this case we should probably catch this before we capture the evidence or if we wait until after we delete the evidence in the case of a failure to avoid orphaned evidence. We're treating the evidence manager as read only (except updating status) and we don't have a way to add unknown evidence into it so having the orphaned evidence around is just going to be confusing.

**Comment by JoelAtDeluxe** [comment]
Addressed in PR #27
